packages/process/src/node/raw-process.ts

```diff
@@ -102,13 +102,13 @@ export class RawProcess extends Process {
             if (this.isForkOptions(options)) {
                 this.process = fork(
                     options.modulePath,
-                    options.args,
-                    options.options);
+                    options.args || [],
```

Comment:
Only this line might be necessary, but it doesn't cost much to provide sane defaults in the other places too.